octobot_commons/tentacles_management/tentacle_requirements.py

```diff
@@ -0,0 +1,94 @@
+#  Drakkar-Software OctoBot-Commons
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+
+class TentacleRequirements:
```

Comment:
Why putting it in Commons instead of Tentacle Manager ?

Reply:
No real reason except for the fact that it's only used in commons caching systems, we could move it to tentacles manager

Reply:
todo:
1. rename into `TentacleRequirementsTree`
2. move into tentacles manager